Gemfile

```diff
@@ -3,13 +3,12 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'smart_proxy', git: 'https://github.com/theforeman/smart-proxy',
-                     branch: 'develop'
-  #gem 'smart_proxy', path: '../smart-proxy'
```

Comment:
This makes it harder for others to consume it. Perhaps you should use an env var that you export to a path? If it's not present, it falls back to the git checkout. Can also make it easier for developers who have a local checkout.